src/layout/header/Menu.tsx

```diff
@@ -59,6 +60,7 @@ const Menu: React.FC<MenuProps> = ({ menuButtonLabel, handleLogout }) => {
   const isPublisher = checkIfPublisher(user);
   const isCurator = checkIfCurator(user);
   const isAppAdmin = checkIfAppAdmin(user);
+  const isInstitutionAdmin = checkIfInstitutionAdmin(user);
```

Comment:
enda en kandidat til ny JIRA-sak: sette disse 4(+) rettighetssjekkene som properties på user-objektet i redux heller enn å kalle hver gang.
va tycks?

Reply:
Bortsett fra AppAdmin ligger disse som roller i user-objektet

Reply:
Forsøker meg på å legge det inn i redux, trenger tilbakemelding!

Reply:
tenkte egentlig det evt kunne vært en egen PR etterpå for det har egentlig ikke noe med NP-558 å gjøre, men greit nok